Demo/Demo/FeatureViewControllers/PayPalDemoViewController.swift

```diff
@@ -6,22 +6,23 @@ class PayPalDemoViewController: FeatureBaseViewController {
 
     // MARK: - UI Components
 
-    lazy var payPalButton: CustomButton = {
-        let payPalButton = CustomButton(title: "PayPal")
-        payPalButton.addTarget(self, action: #selector(didTapPayPalButton), for: .touchUpInside)
-        payPalButton.layer.cornerRadius = 8
-        payPalButton.backgroundColor = .systemBlue
-        payPalButton.tintColor = .white
-        payPalButton.isEnabled = false
-        return payPalButton
-    }()
+    let payPalButton = PayPalButton()
+
+    let payPalCreditButton = PayPalCreditButton()
 
     // MARK: - View Lifecycle & UI Setup
 
     override func viewDidLoad() {
         super.viewDidLoad()
 
+        payPalButton.layer.cornerRadius = 4.0
```

Comment:
What do you think about to keep this view as a lazy var a put this setup inside the closure?

I think is more readable and keep every setup of the view on the same code block and you can keep viewDidLoad more clean. Example:

```
lazy var payPalButton: PayPalButton = {
         let payPalButton = PayPalButton()
         payPalButton.addTarget(self, action: #selector(paymentButtonTapped), for: .touchUpInside)
         payPalButton.layer.cornerRadius = 8
         return payPalButton
 }()
```

Reply:
Yep, that works as well as what we have here. I can update to this since this is more similar to what we had previously. 😄